src/components/auth/Login.js

```diff
@@ -1,11 +1,14 @@
 import { AuthForm } from "./AuthForm";
 import { handleLogin } from "./authHandlers";
+import { useContext } from "react"
```

Comment:
nie jest to wymagane ale dobrze trzymać porządek importów (a najlepiej dodać odpowiednie reguły do eslitna :D) tzn. importy pakietów z package.json jako pierwsze od góry - czyli 
`import { useContext } from "react"` bylby w pierwszej linii w tym przypadku 
a importy z naszej aplikacji na samym dole importów, oddzielone linijką przerwy od importów z package.json.

Łatwiej wtedy na pierwszy rzut oka połapać się co jest użyte w danym komponencie :)